stark-circuits/README.md

```diff
@@ -107,3 +107,20 @@ As a refernece, [to\_bn128](https://github.com/0xEigenLabs/eigen-zkvm/blob/main/
 ## ElementDigest
 
 the struct `ElementDigest` stands for the value of node in Merkle tree. For BN254, each node, including the root, contains 4 Godilocks elements, while BLS12-381 is 6.
+
+## ToDo 
+### Stark Genreation 
+
+- [ ] F5G
```

Comment:
I saw ToDo https://github.com/0xEigenLabs/eigen-zkvm/blob/main/stark-circuits/README.md#todo is already committed, why it become new lines again?